packaging/suse/conf/portus_crono.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=A time-based background job scheduler daemon
+After=apache2.service
```

Comment:
I don't think it required apache2 to be running. @mssola does the code executed by crono talk with Portus via its API? I think it goes straight to the DB. If that's the case we would need to have mariadb up and running, not apache.